modules/index.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Kindly remove these eslint disablement lines and fix eslint error yourself.